dispatcher/conf/magento-proxy.conf

```diff
@@ -16,13 +16,11 @@ LoadModule proxy_module modules/mod_proxy.so
 LoadModule proxy_connect_module modules/mod_proxy_connect.so
 LoadModule proxy_http_module modules/mod_proxy_http.so
 
-Define magento_host http://my-magento.cloud
+Define magento_host http://checkout-y53b4wa-o24f6fbtz7p34.demo.magentosite.cloud
```

Comment:
Same as above - we should have a placeholder string here

Reply:
Arg, yes I will revert that was from my local testing :-)